### ISSUE TITLE
Include a link to squash error information on any system error page

### ISSUE THREAD

**Issue by LynnMcRae** [sul-dlss/argo#524]
Didn't make it a blocker but would really like it if it's as trivial as I believe was said. We want this ability for a new deploy to quickly recognize issues.

**Comment by mejackreed** [comment]
Would be good to make sure from @sul-devops-team @sul-dlss/devops if this is acceptable from a security standpoint?

**Comment by jmartin-sul** [comment]
if there's apprehension around everyone seeing it, what if the link was only shown for fedora admins/"super users" (`user.is_admin`, IIRC)?

**Comment by mejackreed** [comment]
The 500 page should be straight html no ruby. See: https://github.com/sul-dlss/argo/blob/develop/public/500.html

Also shouldn't the admins know what the link is?

**Comment by eefahy** [comment]
I would hesitate to include a direct link to squash in argo app code for the general reason of separating our concerns. For example, squash is a service that's a bit of a pain to maintain and I'm hoping to replace that functionality with splunk or some other log analysis tool in the future.

**Comment by LynnMcRae** [comment]
What we need is the ability to diagnose an error that we or a user encounters easily and quickly, perhaps from information a user can provide us from their screen, to recognize an error as one we already know, without going somewhere to sleuth history and put together clues that might require privileges not all support people will have.  We had this with the old prod, so I consider this a serious functional parity issue.  I appreciate we don't want to run prod apps in debug mode, but we need access to real information about the error when it happens. It's not acceptable that all I or anyone can see now is the same "Sorry, something went wrong" non-information.

**Comment by LynnMcRae** [comment]
DarrenH suggests we rephrase this to express a functional requirement to have basic error information on an Argo version of the "Sorry" page, so I will close this and start a new issue.

**Comment by drh-stanford** [comment]
If we want to develop a customized error page, we can implement a `config.exceptions_app` class that generates an error page that has some information. For example, there is a gem called `exception_handler` that provides for dynamic error page generation.

Right now, we use the Rails 4 default that just returns static HTML for production mode. As far as I know we don't really have the ability to show that same stack trace screen that you see in development mode. In other words, this issue would be new code development, not just a configuration change.

A related problem is that we have in the code base lots of places where we return a 500 status error when it really should be a 4xx error or even a 2xx with a flash notice (i.e., we know what the error is so it's really not a 500 error -- which is a panic error).
